src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# ListBoxOption:
+
+Provides an ListBoxOption component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (node array; required)
+`children` specifies the array of elements that will be displayed within the ListBoxOption component.
+
+### `className` (string; optional)
+`className` specifies Bootstrap class names to apply to the ListBoxOption component. The default is an empty string.
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered ListBoxOption component. The default is div. Example alternatives include li, Button, etc.
+
+### `onOptionSelect` (function, optional)
+`onOptionSelect` is a function that is called when an ListBoxOption component is selected, by mouse or by keyboard, to perform some additional action.
+
+### `other`
+`other` specifies any additional attributes on the ListBoxOption component, which can include aria-labelledby, id, key, etc.
+
+## props that this component needs but should only come from the parent
+
+### `isSelected` (boolean; required)
```

Comment:
I'm not sure this needs to be exposed? Since it's an implementation detail of how `ListBox` will interact with `ListBoxOption`?

Reply:
Yep, thanks for catching that.